css/main.css

```diff
@@ -0,0 +1,1623 @@
+* {
+  font-family: ARS Maquette, Helvetica Neue, Arial, sans-serif;
+}
+
+body {
+  width: 320px;
+}
+
+nav {
+  width: 100%;
+  height: 80px;
+}
+
+.nav-1 {
+  display: none;
+  margin: 0 0 0 0;
+  padding: 0 0 0 0;
+}
+
+.nav-2 .nav-2child {
+  display: none;
+  width: 100%;
+}
+
+.nav-2 .nav-img {
+  margin-left: 90% !important;
+}
+
+.nav-2 {
+  display: flex;
+  background-color: white;
+  width: 100%;
+  height: 30px;
+  justify-content: space-between;
+  text-align: center;
+  padding-top: 10px;
+  border: 1px solid rgb(226, 224, 224);
+}
+
+.search-icon {
+  width: 20px;
+  height: auto;
+}
+
+.section-1 {
+  display: block;
+  margin-bottom: 100px;
+  height: 475px;
+}
+
+.section-1-1 {
+  background-image: url("https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2020/07/kodak-796x448.jpg");
+  background-size: cover;
+  grid-gap: 2px;
+  height: 200px;
+  width: 90%;
+}
+
+.section1-1-heading {
+  font-size: 20px;
+  line-height: 20px;
+  margin-top: 50px;
+  margin-bottom: 25px;
+  font-weight: 700;
+  letter-spacing: -0.84px;
+  color: #fff;
+  font-family: Georgia, 'Times New Roman', Times, serif;
+  gap: normal;
+  position: relative;
+  top: 40px;
+}
+
+.align-date {
+  display: none;
+}
+
+.section1-1-heading-date {
+  color: #fff;
+  font-size: 19px;
+  font-weight: lighter;
+  margin-right: 11px;
+}
+
+.section1-1-heading-title-link {
+  text-decoration: none;
+  color: #fff;
+  position: relative;
+  top: 75px;
+  font-weight: lighter;
+  font-size: 23px;
+  font-family: Georgia, 'Times New Roman', Times, serif;
+}
+
+.media-align-1 {
+  margin-top: -240px;
+}
```

Comment:
- [ ] To fix the text after doing vertical alignment in **mobile** change this margin-top to -150px